python/flask-sqlalchemy/commands/run

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eu
+
+pip3 install --upgrade pip
+
+cd /integration
+
+pip3 install hatch
+hatch clean
+hatch run build:me
+
+cd /app
+
+echo "Installing dependencies"
+pip3 install -r requirements.txt
+pip3 install /integration/dist/* --force-reinstall
+
+echo "Starting app"
+#/commands/processmon processmon.toml
```

Comment:
Processmon doesn't work for this type of app? I'm okay with not using it, but let's remove the comment then.

Reply:
I don't remember why I disabled processmon here (this is copy-pasted from the `flask` test setup, where it is also disabled) -- might be that Flask itself does auto-reload?